Common/Orders/OrderJsonConverter.cs

```diff
@@ -106,8 +106,12 @@ public static Order CreateOrderFromJObject(JObject jObject)
             order.ContingentId = jObject["ContingentId"].Value<int>();
 
             var market = Market.USA;
+
+            //todo: can't safely derive market from security type
             if (securityType == SecurityType.Forex) market = Market.FXCM;
 
+            if (securityType == SecurityType.Crypto) market = Market.Bitfinex;
```

Comment:
'else-if'. Maybe :108 could be the 'else'

Reply:
I'm glad you highlighted this. The class looks "legacy" to me and I hope another dev can confirm if this is obsolete. The practice of deriving a market from a security type doesn't make sense to me.

Reply:
The orders are pretty tricky to decode from JSON so we keep this class around to help people opening JSON results from the GUI. I agree deriving market from sectype is incorrect; we should be able to get that from the symbol object. 

If possible can we remove references to Bitfinex as it not integrated yet and can add it when the brokerage implementation is ready? Anywhere you're forced into having a market like that perhaps it can start with GDAX

Reply:
Yes, I was thinking the market should come from symbol, however, you're not guaranteed to have market, which leaves us back to guessing based on a default, throwing an exception, or leaving it null and hoping nothing blows up down the line. I'm not very familiar of the context in which this is used. Is it just a serialization utility?

Reply:
I think I've removed most (all?) Bitfinex references in the gdax branch, but there's no reason it can't be done here.